pkg/controller/common/cleaner.go

```diff
@@ -57,6 +58,8 @@ func NewCleaner(manager ctrl.Manager, name string, gvrRetriever CleanTypeRetriev
 		retriever:  gvrRetriever,
 		maxRetries: 2,
 	}
+
+	metrics.InitControllerMetrics(c.controllerName())
```

Comment:
Any reason for not adding unit tests for the cleaner metrics?

Reply:
The Clean function itself isn't tested right now so that's probably why.

Reply:
Yep exactly. Funny enough - the `Start` method also isn't called anywhere in the project? Not that Goland can tell anyways

Reply:
Start is done automatically by the controller manager. https://github.com/Azure/aks-app-routing-operator/blob/4af5d634ccab2fc09239b252a37a7e8766ee4741/pkg/controller/common/cleaner.go#L63

Start fulfills an interface needed there